source/calendars/__index.md

```diff
@@ -7,39 +7,37 @@ This calendar lists major events relating to The Apache Software Foundation
 and our many Apache projects.  We also have an automated
 [listing of upcoming Apache-related Meetups][5] (smaller events) available.
 
-This Calendar is maintained by Apache Community Development volunteers (ComDev);
-to have your event listed considered for addition please mail [dev@community.apache.org][1].
-Note that to be listed in this calendar events must <b>already</b> conform
+Apache Community Development (ComDev) volunteers maintain this calendar. To have your event considered for inclusion, email details to [dev@community.apache.org][1].
+We list events that <b>already</b> conform
 to the [Third Party Event Branding Policy][2] and have been approved by
-either the relevant Apache PMC or the Branding Committee.
+the relevant Apache PMC or the Branding Committee.
 
 ## Upcoming Events
 <ul id="events">
  <li><i>List loading</i></li>
 </ul>
 
 <em>Disclaimer: Events called 'ApacheCon' or 'Apache Roadshow' are
-official Apache Software Foundation events. Other listed events are put
-on by projects, or third-party producers, and are listed here to support
-our community, but are not official Foundation events.</em>
+official Apache Software Foundation events. Apache projects or third-party producers put on other listed events; we list them here to support
+our community, but they are not official Foundation events.</em>
 
 <iframe src="https://www.google.com/calendar/embed?src=nerseigospses068jd57bk5ar8%40group.calendar.google.com&ctz=America/New_York"
 style="border: 0" width="800" height="600" frameborder="0" scrolling="no"></iframe>
 
-This Calendar is also available as an [iCal feed][3]
+This calendar is also available as an [iCal feed][3]
 
 <script src="/js/jquery-1.9.1.min.js"></script>
 <script src="https://events.apache.org/js/events-calendar.js"></script>
 <script src="https://apis.google.com/js/client.js"></script>
 
 ## Other Related Events
 
-In addition to major conferences that have applied for approval with us, there are a wide range of other
+In addition to major conferences that have applied for approval with us, there is a wide range of other
 events which feature Apache projects and technologies. A list of upcoming
 [Apache related Meetups is available][5], powered in part by [this script][6]
-(patches welcome!). Secondly, a crowd-sourced list of related events is maintained
+(patches welcome!). Second, a crowd-sourced list of related events is maintained
 as a [Lanyrd Guide][4], and we encourage people to help list
-Apache project related events there.
+Apache-project-related events there.
```

Comment:
Why the dashes? Is this some kind of convention?

Reply:
It's a bit of a moot point - we should be removing any mention of Lanyrd, as that site is long-gone.